src/bin/cql-stress-cassandra-stress/operation/row_generator.rs

```diff
@@ -78,6 +80,41 @@ pub struct RowGenerator {
     pk_seed_distribution: Arc<dyn Distribution>,
     pk_generator: Generator,
     column_generators: Vec<Generator>,
+    // Map column name to the index of generated value in resulting vector.
+    #[cfg(feature = "user-profile")]
+    index_map: HashMap<String, usize>,
+}
+
+impl RowGenerator {
+    pub fn new(
+        pk_seed_distribution: Arc<dyn Distribution>,
+        pk_generator: Generator,
+        column_generators: Vec<Generator>,
+    ) -> Self {
+        #[cfg(feature = "user-profile")]
+        let index_map = {
+            let mut index_map = HashMap::new();
+
+            index_map.insert(pk_generator.get_col_name().to_owned(), 0);
+            for (i, gen) in column_generators.iter().enumerate() {
+                index_map.insert(gen.get_col_name().to_owned(), i + 1);
+            }
+            index_map
+        };
+
+        Self {
+            pk_seed_distribution,
+            pk_generator,
+            column_generators,
+            #[cfg(feature = "user-profile")]
+            index_map,
+        }
+    }
+
+    #[cfg(feature = "user-profile")]
+    pub fn index_of(&self, name: &str) -> Option<usize> {
+        self.index_map.get(name).copied()
+    }
```

Comment:
Perhaps a more explicit name? E.g. `statement_index_of_column_with_name`, but maybe you can come up with something more concise.